GAE/src/org/waterforpeople/mapping/dataexport/ExportUtils.java

```diff
@@ -0,0 +1,97 @@
+
+package org.waterforpeople.mapping.dataexport;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+
+import com.gallatinsystems.common.util.StringUtil;
+
+public class ExportUtils {
```

Comment:
This is a very pedantic comment from a readability perspective this is used for import as well but the name suggests otherwise.